src/Screens/HomeScreen/styles.ts

```diff
@@ -1,10 +1,11 @@
 import { StyleSheet,Dimensions } from "react-native";
+import { colors } from "../../assets/arrays/theme";
 
 const styles = StyleSheet.create({
     container:{
        width:'100%',
        height: Dimensions.get('window').height,
```

Comment:
you can export dimension data from a metrics file inside the theme directory.
eg.
```
Basic usage:

export const screenWidth = Dimensions.get('window').width;
export const screenHeight = Dimension.get('window').height;

Reduced syntax

const { width: screenWidth, height: screenHeight } = Dimensions.get('window');
export { screenWidth, screenHeight }

```

then simply use **screenWidth**, **screenHeight** through out the app